### ISSUE TITLE
Full chest destroys payment / still gives item. {1.4.0 Beta}

### ISSUE THREAD

**Issue by waffleston** [SparklingComet/TradeShop#4]
```
Using: git-spigot-ede465d-bb4ae3b (MC: 1.12)
```
When a chest has no open spots for payment, the payment item is lost.  

Example:
```
[Trade]
1 stone
64 cobblestone
```
The chest is filled to the brim with 27 stacks of 64 stone.  A buyer approaches the chest and performs a trade.  The player is deducted 64 cobblestone and given one stone.  The chest, however, only deducts one stone and does not get the payment of 64 cobblestone.  

---
No error is thrown in the console, and the chat outputs the standard trade success message.

**Comment by SparklingComet** [comment]
This was unknown to me but should be easy to fix. You may expect a patch for this this week. (Provided, of course, that I can reproduce the issue first)

**Comment by KillerOfPie** [comment]
@SparklingComet Never mind read that wrong, still know what is causing it though.
*Edited from original comment

**Comment by SparklingComet** [comment]
@KillerOfPie Is this still a thing or was it fixed?

// EDIT:
Closed by mistake.

**Comment by SparklingComet** [comment]
Fixed in @KillerOfPie 's PR (#6)
Thank you both very much.
